examples/sparse/hgnn.py

```diff
@@ -0,0 +1,95 @@
+"""
+Hypergraph Neural Networks (https://arxiv.org/pdf/1809.09401.pdf)
+"""
+import dgl
+import dgl.data
+import dgl.mock_sparse
```

Comment:
Is it necessary to import `dgl.data` and `dgl.mock_sparse`?

Reply:
No need to import `dgl.data`, but `dgl.mock_sparse` will throw an error without explicit importing.

Reply:
I see. Perhaps either `from dgl.mock_sparse import xx` or alternatively `import dgl.mock_sparse as xx`. The existing examples follow the former option.